apps/web/src/ui/pool/ConcentratedLiquidityRemoveWidget.tsx

```diff
@@ -566,6 +590,17 @@ export const ConcentratedLiquidityRemoveWidget: FC<
                     )}
                   </List.Control>
                 </List>
+                {positionHasNativeToken ? (
+                  <div className="flex items-center justify-between">
+                    <span className="text-sm text-muted-foreground">
+                      {`Receive ${nativeToken.wrapped.symbol} as ${nativeToken.symbol}`}
```

Comment:
By default, users should receive the unwrapped native token.
That's why the toggle state is called `receiveWrapped` and the default state is off.

![CleanShot 2025-01-06 at 12 31 04@2x](https://github.com/user-attachments/assets/8904f687-6747-43af-8ee2-b4bc1d8469f0)

However, do we think the text `Receive WETH as ETH` is clear enough to convey which currency they will get?
If it was `Receive WETH instead of ETH`, I think it would be easier to understand.

Reply:
Agreed. `instead of` is clearer.

Reply:
Changed to `instead of`